lib/proxyquire.js

```diff
@@ -4,6 +4,7 @@
 var Module = require('module')
 var resolve = require('resolve')
 var dirname = require('path').dirname
+var pathResolve = require('path').resolve
```

Comment:
Much as it'll be a bit disruptive I think we should just do `var path = require('path')` and rename other "path" names to be more descriptive. I'd rather someone see `path.*` especially for fns that aren't obviously named (resolve).

Reply:
Yeah, I was thinking the same thing.  I'll do this.